main.sh

```diff
@@ -1,4 +1,11 @@
 #!/bin/bash
 set -e
 playwright install
-python main.py
+python main.py --dataset 1MB-chunks --zarr-version v2 --action zoom_in --zoom-level 4 --s3-bucket s3://carbonplan-benchmarks
```

Comment:
should we enable the `--detect-provider` flag so we can track where the results are recorded from?

Reply:
good call, thanks!